src/chrome/content/rules/Giphy.com.xml

```diff
@@ -1,77 +1,63 @@
 
 <!--
-	CDN buckets:
-
-		- giphygifs.s3.amazonaws.com
-
-			- media10
-
-		- d16jfyletng9p5.cloudfront.net
-
-
-	Problematic subdomains:
-
-		- media10 ²
-		- www		(refused)
-
-	² AmazonWS
-
-
-	Fully covered subdomains:
-
-		- (www.)?	(www → ^)
-		- api
-		- i
-		- media
-		- media10	(→ giphygifs.s3.amazonaws.com)
-		- media\d+	(0-4)
-
-
 	Insecure cookies are set for these domains:
 
 		- .giphy.com
 
+	Invalid certificate:
+		blog.giphy.com
 
-	Mixed content:
+	Time out:
+		happyholidays.giphy.com
+		labs.giphy.com
+		twitter.giphy.com
 
-		- Images on ^ from media4.giphy.com *
+	Refused:
+		gj.giphy.com
 
-	* Secured by us
+	No working URL known:
+		i.giphy.com
+		outlook.giphy.com
+		pingback.giphy.com
```

Comment:
Well this host is just a API that listen on POST requests. If Travis allow us to just add it I would like to keep it, because this API was secured with the wildcard.